### ISSUE TITLE
Indentation reporting not working as it used to

### ISSUE THREAD

**Issue by paulGeoghegan** [nvaccess/nvda#15807]
### Steps to reproduce:
Install the latest version of NVDA,
Open NVDA settings,
Go to document formatting,
Set line indentation reporting to speach and then press apply.
### Actual behavior:
When I am writing code the line indentation for a line is reported however when I navigate using the arrow keys the line indentation isn't reported again until I reach a line that has a different indentation level. This worked for me previously but no longer works on the latest version of NVDA (nvda_2023.3).
This occurs in VS Code and notepad as far as I have tested.
### Expected behavior:
NVDA should report the indentation level for every line irrespective of whether or not it is the same.
### NVDA logs, crash dumps and other attachments:
NA
#### NVDA installed/portable/running from source:
Install
#### NVDA version:
nvda_2023.3
#### Windows version:
11
#### Name and version of other software in use when reproducing the issue:
NA
#### Other information about your system:
NA
#### Does the issue still occur after restarting your computer?
Yes
#### Have you tried any other versions of NVDA? If so, please report their behaviors.
No
#### If NVDA add-ons are disabled, is your problem still occurring?
Yes
#### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?
Yes

**Comment by XLTechie** [comment]
CC @SamKacer

**Comment by jcsteh** [comment]
I'm fairly sure report indentation always only reported the indentation when it changed for efficiency reasons. If you turn off reporting of indentation, tab characters will be reported always if your punctuation level is set to all, but that's because it's a specific character, not because of indentation.

**Comment by jcsteh** [comment]
At least, that's how it worked when it was first implemented. Maybe it regressed at some point and then reverted to its original state in 2023.3?

**Comment by XLTechie** [comment]
I agree with @jcsteh--what you describe appears to be the normal behavior.

I entered the following in notepad:
```
    This
    is a
  test

  of
  indentation

```

Here is what Speech Logger captures when reading that line by line in NVDA 2023.1:
```
4 space  This
is a
2 space  test
no indent  blank
2 space  of
indentation
no indent  blank
```

And here is the result of reading that in NVDA 2023.3:
```
4 space  This
is a
2 space  test
no indent  blank
2 space  of
indentation
no indent  blank
```

I can detect no difference between the two.

I CCed @SamKacer, because I thought this might have been a result of the new #13394 behavior in 2023.3 (I.e. the "Ignore blank lines for line indentation reporting" option). However upon re-reading the issue description, I doubt that feature is involved. To test this, I turned that option on, and now the result is:

```
4 space  This
is a
2 space  test
blank
of
indentation
blank
```

In any event, what you describe is the expected behavior, and how NVDA has behaved for years.
